src/js/modules/i18n/infragistics.ui.zoombar-ru.js

```diff
@@ -9,14 +9,13 @@
 
 (function (factory) {
 	if (typeof define === "function" && define.amd) {
-		define( [], factory );
+		define( ["jquery"], factory );
 	} else {
-		return factory();
+		factory(jQuery);
```

Comment:
There should be a `return` because if the factory isn't returning, then doing  

`var locale = require("path/to/locale");`

will result in `undefined`